scripts/pre-commit

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
Shouldn't we include a copyright header here as well?

Reply:
done